README.md

```diff
@@ -1,18 +1,16 @@
-# Google Cloud Platform plugin for Cloudera Director
+# Google Cloud Platform plugin for Cloudera Altus Director
 [![Build Status](https://api.travis-ci.org/cloudera/director-google-plugin.svg)](https://travis-ci.org/cloudera/director-google-plugin)
 
-The Cloudera Director Google Plugin is an implementation of the [Cloudera Director Service Provider Interface](https://github.com/cloudera/director-spi) for the [Google Cloud Platform](https://cloud.google.com). It presently supports provisioning Google Compute Engine resources and will soon expand to include support for provisioning Google Cloud SQL resources.
-
-These instructions describe how to configure and exercise the Cloudera Director Google Plugin.
```

Comment:
Looks like this paragraph disappeared.

Reply:
yup. you are right. I'll add that and try again